dipy/tracking/_utils.py

```diff
@@ -60,7 +60,7 @@ def _to_voxel_coordinates(streamline, lin_T, offset):
     raises an error for negative voxel values."""
     inds = np.dot(streamline, lin_T)
     inds += offset
-    if inds.min() < 0:
+    if inds.min().round(decimals=6) < 0:
         raise IndexError('streamline has points that map to negative voxel'
```

Comment:
Oh yeah - this PR is also crucial for the release. We need a test!

On Mon, Oct 26, 2015 at 7:28 AM, Eleftherios Garyfallidis <
notifications@github.com> wrote:

> In dipy/tracking/_utils.py
> <https://github.com/nipy/dipy/pull/678#discussion_r42999436>:
>
> > @@ -60,7 +60,7 @@ def _to_voxel_coordinates(streamline, lin_T, offset):
> >      raises an error for negative voxel values."""
> >      inds = np.dot(streamline, lin_T)
> >      inds += offset
> > -    if inds.min() < 0:
> > +    if inds.min().round(decimals=6) < 0:
> >          raise IndexError('streamline has points that map to negative voxel'
>
> Ignore this. Not important.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/678/files#r42999436>.
>